meta-facebook/yv35-cl/src/platform/plat_sensor_table.c

```diff
@@ -415,6 +439,10 @@ void pal_extend_sensor_config()
 	for (int index = 0; index < sensor_count; index++) {
 		ltc4286_sensor_config_table[index].init_args = &ltc4286_init_args[arg_index];
 	}
+	sensor_count = ARRAY_SIZE(ltc4282_sensor_config_table);
```

Comment:
It seems that the for loops like in line 443 could be consolidated with the case statements below (such as line 467) to make this code more efficient.  This would reduce at least once extra trip through the sensor indexes if they are combined for each HSC module type.

Reply:
Thanks for your suggestion.
The code is forced updated.
Please review it again.
Thanks.